tests/constraint/test_TSR.cpp

```diff
@@ -18,15 +18,15 @@ using dart::common::make_unique;
 using DefaultRNG = RNGWrapper<std::default_random_engine>;
 
 
-static Eigen::Vector6d getPose(Eigen::Isometry3d isometry)
+static inline Eigen::Vector6d getPose(Eigen::Isometry3d isometry)
```

Comment:
From the documentation:
**-Wunused-function**
Warn whenever a static function is declared but not defined or a non-inline static function is unused. This warning is enabled by -Wall. 
In our case a non-inline static function's not used.